unitylibs/core/features/progress-circle/progress-circle.js

```diff
@@ -0,0 +1,25 @@
+import { createTag } from '../../../scripts/utils.js';
+
+export default function createProgressCircle(cfg) {
+  const { unityEl, targetEl, progressCircleEvent } = cfg;
+  const pdom = `<div class="spectrum-ProgressCircle-track"></div>
+  <div class="spectrum-ProgressCircle-fills">
```

Comment:
Can we make it a constant outside the function

Reply:
Fixed